src/Interfaces/Standalone/CJ77.F90

```diff
@@ -60,14 +61,15 @@ program standalone
 
       write(6,'(a)') '>>>begin Cloud-J v7.7 Standalone'
 
+      RC = CLDJ_SUCCESS
       NLEVELS = 57
       ANU = AN_
       JVNU = JVN_
       L1U = L1_
       amIRoot = .true.
 !---read in & store all fast-JX data:   single call at set up
 !-----------------------------------------------------------------------      
-      call INIT_CLDJ (amIRoot,'./tables/',NLEVELS,TITLJXX,JVNU,NJXX)
+      call INIT_CLDJ (amIRoot,'./tables/',NLEVELS,TITLJXX,JVNU,NJXX,RC)
```

Comment:
We need to call CLOUDJ_ERROR and return if RC returns failure.  Ditto for all other subroutine calls returning RC.